torch/_inductor/utils.py

```diff
@@ -1048,3 +1049,14 @@ def is_dynamic(*args):
             raise ValueError(f"unexpected type for is_dynamic {type(t)}")
 
     return False
+
+
+# Placeholder strings used in triton codegen.
+class Placeholder(enum.Enum):
+    # The placeholder for the actual name of a triton kernel.
+    # e.g. for "def triton_" it would be "triton_"
+    KERNEL_NAME = "KERNEL_NAME"
+
+    # The descriptive name of the triton kernel; when unique_kernel_names = False, this
+    # placeholder will be replaced with a string with more information.
+    DESCRIPTIVE_KRNL_NAME = "DESCRIPTIVE_KRNL_NAME"
```

Comment:
full words krnl->kernel

Reply:
@jansel I thought it would be good if none of these names are substrings of other names (i.e. KERNEL_NAME and DESCRIPTIVE_KERNEL_NAME), so that we don't have any issues with the order we replace them in?

Although it's fine as long as we search/replace DESCRIPTIVE_KERNEL_NAME before KERNEL_NAME.

https://github.com/pytorch/pytorch/blob/01b662bafe54dfda561d442015dc512edf8b1564/torch/_inductor/codegen/triton.py#L2503-L2504

Reply:
Lets just change the search/replace terms to match then.

Reply:
Changed it to DESCRIPTIVE_NAME and KERNEL_NAME